.github/workflows/bi-connectors.yml

```diff
@@ -11,40 +11,41 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Pack Tableau JDBC connector
       id: pack-tableau-jdbc
       run: |
         zip -r opensearch_sql_jdbc.taco . -x *.taco
       working-directory: bi-connectors/TableauConnector/opensearch_sql_jdbc
     - name: Prepare Power BI ODBC connector
       run: |
-        cp OpenSearch.pq OpenSearch.m
+        cp OpenSearchProject-ODBC.pq OpenSearchProject-ODBC.m
```

Comment:
Based on the commit description. This should be just "OpenSearchProject" excluding the -ODBC.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008